packages/pagination/examples/basic.md

```diff
@@ -93,3 +95,181 @@ initialState = {
+
+  return (
+    <Grid>
+      <Row alignItems="center">
+        <Col alignSelf="center" className="u-mb u-mt">
+          <FlexContainer>
+            {circles.map((circle, index) =>
+              index === cursor ? <CircleFill key={index} /> : <CircleStroke key={index} />
+            )}
+          </FlexContainer>
+          <VisuallyHidden aria-live="polite">{`The cursor is ${cursor}`}</VisuallyHidden>
+          <CursorPagination aria-label="Cursor pagination">
+            <CursorPagination.Previous onClick={onPrevious} disabled={cursor === 0}>
+              Previous
+            </CursorPagination.Previous>
+            <CursorPagination.Next onClick={onNext} disabled={cursor === circles.length - 1}>
+              Next
+            </CursorPagination.Next>
+          </CursorPagination>
+        </Col>
+      </Row>
+    </Grid>
+  );
+};
+
+<CursorPaginationExample />;
+```
+
+#### First, Previous, Next, & Last
+
+```jsx
+const { Well } = require('@zendeskgarden/react-notifications/src');
+const { getColor } = require('@zendeskgarden/react-theming/src');
+
+const StyledCircle = styled.div`
+  margin: 0 ${props => props.theme.space.base}px;
+  border-radius: 100%;
+  height: ${props => props.theme.space.base * 3}px;
+  width: ${props => props.theme.space.base * 3}px;
+`;
+
+const CircleStroke = styled(StyledCircle)`
+  border: ${props =>
+    `0.6px ${props.theme.borderStyles.solid} ${getColor('grey', 300, props.theme)}`};
+`;
+
+const CircleFill = styled(StyledCircle)`
+  background: ${props => getColor('blue', 600, props.theme)};
+`;
+
+const FlexContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  margin: ${props => `${props.theme.space.base * 5}px 0`};
+`;
+
+const VisuallyHidden = styled.div`
+  border: 0;
+  clip: rect(0 0 0 0);
+  height: 1px;
+  margin: -1px;
+  overflow: hidden;
+  padding: 0;
+  position: absolute;
+  width: 1px;
+  white-space: nowrap;
+  word-wrap: normal;
+`;
+
+const CursorPaginationExample = () => {
+  const [cursor, setCursor] = React.useState(0);
+  const [startLast, setStartLast] = React.useState(true);
+  const circles = [0, 1, 2, 3, 4];
+  const onNext = () => {
+    if (cursor < circles.length - 1) {
+      setCursor(cursor + 1);
+    }
+  };
+  const onPrevious = () => {
+    if (cursor > 0) {
+      setCursor(cursor - 1);
+    }
+  };
+  const onFirst = () => setCursor(0);
+  const onLast = () => setCursor(circles.length - 1);
+
+  return (
+    <Grid>
+      <Row alignItems="center">
+        <Col alignSelf="center" className="u-mb u-mt">
+          <FlexContainer>
+            {circles.map((circle, index) =>
+              index === cursor ? <CircleFill key={index} /> : <CircleStroke key={index} />
+            )}
+          </FlexContainer>
+          <VisuallyHidden aria-live="polite">{`The cursor is ${cursor}`}</VisuallyHidden>
+          <CursorPagination aria-label="Cursor pagination">
+            <CursorPagination.First onClick={onFirst} disabled={cursor === 0}>
```

Comment:
`CursorPagination.First`, `CursorPagination.Previous`, `CursorPagination.Next`, and `CursorPagination.Last` are rendered as `<button>` elements within a `<nav>`. I often render a list in a `nav` but decided on using buttons.

> A nav element doesn't have to contain a list, it can contain other kinds of content as well.

https://html.spec.whatwg.org/multipage/sections.html#the-nav-element

Reply:
I think this makes sense. Although, we may want to think about reconciling the unexpected structural difference between `Pagination` and `CursorPagination`.